docs/spire.md

```diff
@@ -6,13 +6,58 @@ weight: 1660
 -->
 ⚠️ This is a work in progress: SPIRE support is not yet functional
 
-TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE. See TEP-0089 for details on the overall design and feature set.
+TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE and enabling TaskRun to sign and verify the TaskRunStatus. See TEP-0089 for details on the overall design and feature set.
```

Comment:
Add a link for users who may not know where to find TEPs:

```suggestion
TaskRun result attestations is currently an alpha experimental feature. Currently all that is implemented is support for configuring Tekton to connect to SPIRE and enabling TaskRun to sign and verify the TaskRunStatus. See [TEP-0089] for details on the overall design and feature set.

 [TEP-0089]: https://github.com/tektoncd/community/blob/main/teps/0089-nonfalsifiable-provenance-support.md
```

Reply:
+1. 
Added a link to the TEP-0089 string below.